semgrep/semgrep/rule_schema.yaml

```diff
@@ -291,6 +291,9 @@ properties:
         metadata:
           title: Arbitrary structured data for your own reference
           type: object
+        settings:
```

Comment:
We should include a test case (maybe `x = 2; x == 2`, and check that with constant prop off this doesn't match)

Reply:
Note that eb54e024f0c9f3dcac854e24e37a988bc775f6b4 added that test to semgrep-core/tests.

Reply:
Would still be good to have one for semgrep, but that's good